skedify/sdk-js
# feature/SKED-5878

> See JIRA https://skedify.atlassian.net/browse/SKED-5878

Currently, semantic-release only releases for `feat`, `fix` & `perf` type of commits. I've added a .releaserc file to make sure we also create releases on `build` & `test` commits.

**How to test**
* check out the master branch
* Merge this branch in master locally: `git merge feature/SKED-5878` - the master branch is used to make release
* remove "@semantic-release/npm" & "@semantic-release/github" from the .releaserc file, we only want to see the commits
* Run the semantic-release: `npm run semantic-release` (this will do a dry-run, so nothing will be released)
* You should see that the commits are analysed and that a commit of type `test` should state the following:
```shell
[semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: test: allow to disable access tokens mocks in tests
[semantic-release] [@semantic-release/commit-analyzer] › ℹ  The release type for the commit is patch
```

* Remove the releaseRules from the `.releaserc` file (or delete the complete file).
* The message should be `The commit should not trigger a release`
